### ISSUE TITLE
Fix automatically generated end-to-end test

### ISSUE THREAD

**Issue by adrianbranescu** [Adrian-s-JavaScript-Programs/angular-locomotive-check-digit-generator#2]
When creating the project using Angular CLI, it automatically generated one default end-to-end test which fails when running "ng e2e" command. The test is located inside "e2e\src\app.e2e-spec.ts" file.

The application itself works fine, but the automatically generated end-to-end test needs manual correction.

![Console1](https://user-images.githubusercontent.com/59761765/104499123-46e1b180-55e5-11eb-8a38-41c23655c714.png)
![Console2](https://user-images.githubusercontent.com/59761765/104499127-4812de80-55e5-11eb-9068-a2d707359148.png)

**Comment by adrianbranescu** [comment]
When the project was created (using "ng new locomotive-check-digit-app" command), a default "app.component.html" file was generated, which contained "locomotive-check-digit-app app is running!" message.

Also, default files containing unit tests and end-to-end tests were generated. Some of those tests were verifying if "locomotive-check-digit-app app is running!" message was being displayed (one unit test and one end-to-end test).

The problem is that I modified the application and that default message is no longer being displayed, because my application was intended to display other content to the user.

When solving Issue #1, I removed the unit test that verified if that message was being displayed.

However, a similar end-to-end test remained in "app.e2e-spec.ts" file.
That test should also be deleted, since it was supposed to verify something that is no longer being used.

In "app.e2e-spec.ts" this whole block should be removed:
```ts
  it('should display welcome message', () => {
    page.navigateTo();
    expect(page.getTitleText()).toEqual('locomotive-check-digit-app app is running!');
  });
```
